src/main/java/Glendon.java

```diff
@@ -0,0 +1,117 @@
+import java.util.Scanner;
+
+public class Glendon {
+    public static final int maxNumberOfTask = 100;
+    public static Task[] list = new Task[maxNumberOfTask];
+    public static int taskCounter = 0;
+
+    public static void main(String[] args) {
+        printWelcomeMessage();
+
+        Scanner in = new Scanner(System.in);
+        String response = in.nextLine();
+
+        while (response != null) {
+            switch (response.split(" ")[0]) {
+            case "bye":
+                printBye();
+                response = null;
+                break;
+            case "list":
+                printList();
+                response = in.nextLine();
+                break;
+            case "mark":
+                markDone(response);
+                response = in.nextLine();
+                break;
+            case "unmark":
+                unmarkDone(response);
+                response = in.nextLine();
+                break;
+            case "todo":
+                addTodo(response);
+                response = in.nextLine();
+                break;
+            case "deadline":
+                addDeadline(response);
+                response = in.nextLine();
+                break;
+            case "event":
+                addEvent(response);
+                response = in.nextLine();
+                break;
+            default:
+                response = in.nextLine();
+                break;
+            }
+        }
+    }
+
+    private static void addEvent(String response) {
+        String[] answers = response.split("/");
+        list[taskCounter] = new Event(answers[0].substring(6), answers[1].substring(5), answers[2].substring(3));
+        printAddedTask();
+        taskCounter++;
+    }
+
+    private static void printAddedTask() {
+        System.out.println("Got it. I've added this task:");
+        System.out.println(list[taskCounter]);
+        System.out.println("Now you have " + (taskCounter+1) + " tasks in the list.");
+    }
+
+    private static void addDeadline(String response) {
+        String[] answers = response.split("/");
+        list[taskCounter] = new Deadline(answers[0].substring(9), answers[1].substring(3));
+        printAddedTask();
+        taskCounter++;
+    }
+
+    private static void addTodo(String response) {
+        list[taskCounter] = new Todo(response.substring(5));
+        printAddedTask();
+        taskCounter++;
+    }
+
```

Comment:
Perhaps could handle case where wrong input is detected and other edge cases, but this is follow up from necessary tasks.